### DISCUSSION TITLE
Problem getting forecast

### DISCUSSION THREAD

**Discussion by AGPennypacker** [Gulivertx/cbatmo#65] [General]
Im having problem showing forecast. I get an error invallid API key, 401. When i look at my page on openweather API page i have a key there and i did copy it to my .env file. I also tried to generate a new API key with same result.
I have tried formating OPENWEATHER_API_KEY in different ways and non is working.

line in my .env file looks likt this

OPENWEATHER_API_KEY=api.openweathermap.org/data/2.5/forecast?lat=61.86297&lon=16.018633&appid=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 

All the x are my API key from open weather site.

What am i doing wrong?

**Comment by Gulivertx** [comment]
Hi,
You must only add api key without the url, just keep your key xxxxxxxx. 
OPENWEATHER_API_KEY=xxxxxxxxxxxxxxxxxxxx

**Reply by AGPennypacker** [reply]
Thanks for quick reply, now it works!

By the way thanks for a really good APP!

**Reply by Gulivertx** [reply]
Nice to have a good feedback 😊
